Team.html

```
@@ -0,0 +1,366 @@
+                    <div class="description">
+                        <h2>Shenuka Dias</h2>
+                        <p>Student Role 4</p>
+                        <p>Pages developed</p>
+                        <p>Profile Page | Sitemap | Content page</p>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+    <!-- Footer Section -->
+    <footer class="section footer">
```

HTMLHint (1.5.0)

[error] Tag must be paired, missing: [ </footer> ], start tag match failed [ <footer class="section footer"> ] on line 251.

(tag-pair)